middleware/package.json

```diff
@@ -12,7 +12,7 @@
     "prepack": "npm run build",
     "lint": "tslint --project . --format stylish",
     "format": "find src -name '*.ts' | xargs clang-format --style=file -i",
-    "test": "npm run build && ava 'build/**/*-test.js'"
+    "test": "npm run build && ava build/**/*-test.js"
```

Comment:
I think the quotes here are important, if the user doesn't have globstar (e.g. macOS, which has an old version of bash), then the `**` won't work. So the quotes mean ava will handle the glob.

Reply:
Interesting. Could you verify that? Because the quotes actually broke the glob on my OSX machine after the update for ava.

Reply:
Maybe ava removed glob support. And maybe it works anyway because if globstar is disabled, ** should work the same as *. Try running `shopt -s globstar` and it should fail on mac OS (because macOS bash is stuck at 3 for licensing reasons and globstar was added in 4.0 https://www.tldp.org/LDP/abs/html/bashver4.html). Anyway, if it's working, that's fine. Maybe just replace the ** with a * and see if it still works?

Reply:
Done